cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tether/PeerMetadata.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.tether;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Metadata about a tethered peer.
+ */
+public class PeerMetadata {
+  private final List<NamespaceAllocation> namespaces;
+  private final Map<String, String> metadata;
+
+  public PeerMetadata(List<NamespaceAllocation> namespaces, Map<String, String> metadata) {
```

Comment:
What does metadata mean in this case?

Reply:
It's the metadata associated with the peer. ex: project, region when peer is located.
Added a comment to clarify